src/imagery/i.saocom/i.saocom.geocode/i.saocom.geocode.html

```diff
@@ -0,0 +1,49 @@
+<h2>DESCRIPTION</h2>
+
+The <em>i.saocom.geocode</em> module allows allows the projection of SAOCOM-1 L1A raw or derived products from radar coordinates to a cartographic coordinate  system (CS). This can be applied either to bands imported by <em>i.saocom.import</em> and to any raster map calculated from these inputs, or directly to the SAOCOM original files. The tool does not currently support the geocoding of image subsets, so the whole extent must be processed.
+
+<h2>NOTES</h2>
+
+When the tool is directly run on the original SAOCOM files, it will internally call <em>i.saocom.import</em> to import the real and imaginary bands to a temporary XY unprojected location. In this case the <b>data</b> option must be used. If the user wants to geocode already imported files, the <b>map</b> option must be specified. The tool will not allow the use of both options, either one of them must be selected.
+
+This module makes use of the <b>GCPS.csv</b> file generated by <em>i.saocom.import</em> and associated to a specific image. This GCP file is associated to the image basename specified in the option <b>basename</b>.
+
+The tool will write the output to a Geotiff external file in the directory specified in the <b>dbase</b> option. The output files will be projected to the CS of the target location indicated in the option <b>location</b>. If the specified location does not exits, GRASS will create it with the CS EPSG:4326. The output map name will we the same as the input map, with the suffix <em>_geo</em> added at the end.
+    
+
+<h2>EXAMPLES</h2>
+
+<h3>Within an XY unprojeted location where the SAOCOM-1 bands have already been imported and processed, create an amplitude image from the real and imaginary bands, geocode it and then import it to a location called <em>saocom_geo</em></h3>
+
+<div class="code"><pre>
+#Calculate the amplitude image 
+i.sar.amplitude real='SAO1B_20211222_hh_real' imag='SAO1B_20211222_hh_imag' output='SAO1B_20211222_hh_amp' 
+
+#Geocode it and import it to new location. The external temporary raster will be saved as GeoTiff in the $HOME directory
+i.saocom.geocode map='SAO1B_20211222_hh_amp' basename='SAO1B_20211222' dbase=$HOME location='saocom_geo'
```

Comment:
```suggestion
i.saocom.geocode map=SAO1B_20211222_hh_amp basename=SAO1B_20211222 dbase=$HOME location=saocom_geo
```

What would then be the CRS of the `saocom_geo` location?

Reply:
It can be any CRS if `saocom_geo` already exists. In case it does not exist, it will create it and assign the CRS EPSG:4326